src/type_translator.ts

```diff
@@ -336,8 +336,7 @@ export class TypeTranslator {
         }
         return this.symbolToString(type.symbol, true);
       case ts.TypeFlags.ESSymbol:
-        // NOTE: currently this is just a typedef for {?}, shrug.
-        // https://github.com/google/closure-compiler/blob/55cf43ee31e80d89d7087af65b5542aa63987874/externs/es3.js#L34
+      case ts.TypeFlags.UniqueESSymbol:
```

Comment:
Maybe put a comment in here about how these are different?

Reply:
Done.